src/plots/cartesian/autorange.js

```diff
@@ -371,16 +374,22 @@ function findExtremes(ax, data, opts) {
         if(!isNumeric(di)) return;
         ppadiplus = ppadplus(i);
         ppadiminus = ppadminus(i);
-        vmin = di - vpadminus(i);
-        vmax = di + vpadplus(i);
-        // special case for log axes: if vpad makes this object span
-        // more than an order of mag, clip it to one order. This is so
-        // we don't have non-positive errors or absurdly large lower
-        // range due to rounding errors
-        if(isLog && vmin < vmax / 10) vmin = vmax / 10;
-
-        dmin = ax.c2l(vmin);
-        dmax = ax.c2l(vmax);
+
+        if(vpadLinearized) {
+            dmin = ax.c2l(di) - vpadminus(i);
+            dmax = ax.c2l(di) + vpadplus(i);
+        } else {
+            vmin = di - vpadminus(i);
+            vmax = di + vpadplus(i);
+            // special case for log axes: if vpad makes this object span
+            // more than an order of mag, clip it to one order. This is so
+            // we don't have non-positive errors or absurdly large lower
+            // range due to rounding errors
+            // if(isLog && vmin < vmax / 10) vmin = vmax / 10;
```

Comment:
What happens when you uncomment this line?

Reply:
It seems to be an error. Fixed in commit 1876e8c5caaad78ca7d50fa0282c8e1c780b9d00
(Maybe I forgot to uncomment this line while checking its behaviour)